lib/iris/util.py

```diff
@@ -1,4 +1,4 @@
-# (C) British Crown Copyright 2010 - 2016, Met Office
+# (C) British Crown Copyright 2010 - 2017, Met Office
```

Comment:
I made some changes, then didn't need them + reverted them all in a subsequent commit.
But then you can't revert the date without the licence check failing.
I think you *can* remove this when you squish.